src/main/java/net/dv8tion/jda/core/handle/PresenceUpdateHandler.java

```diff
@@ -208,11 +208,9 @@ protected Long handleInternally(JSONObject content)
 
             //If the OnlineStatus is OFFLINE, ignore the event and return.
             OnlineStatus status = OnlineStatus.fromKey(content.getString("status"));
-            if (status == OnlineStatus.OFFLINE)
```

Comment:
We lost the ability to return in an `OFFLINE` non-guild situation.
Is that correct handling?

Reply:
Can you elaborate? Not sure what you mean

Reply:
Nevermind. I see what has changed here. The fast escape logic is no longer needed.